pkg/alerter/alerter.go

```diff
@@ -0,0 +1,71 @@
+package alerter
+
+import (
+	"fmt"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/sirupsen/logrus"
+)
+
+type Alerter struct {
+	Alerts   Alerts
+	AlertsCh chan Alerts
+}
+
+func NewAlerter() *Alerter {
+	alerts := make(Alerts)
+	alertsCh := make(chan Alerts)
+
+	go func() {
+		defer close(alertsCh)
+		for alert := range alertsCh {
+			for k, v := range alert {
+				if val, ok := alerts[k]; ok {
+					alerts[k] = append(val, v...)
+				} else {
+					alerts[k] = v
+				}
+			}
+		}
+	}()
+
+	return &Alerter{
+		Alerts:   alerts,
+		AlertsCh: alertsCh,
+	}
+}
+
+func (a *Alerter) SendAlert(key string, alert Alert) {
+	a.AlertsCh <- Alerts{
+		key: []Alert{alert},
+	}
+}
+
+func (a *Alerter) IgnoreResources(resources []resource.Resource) []resource.Resource {
```

Comment:
thought we decided to move this logic in analyzer ?

Reply:
Indeed, done in 442d3ed